packages/contracts/tests/unit.ts

```diff
@@ -133,3 +127,53 @@ describe('zkShuffle Unit Test', function () {
     });
 
 });
+
+describe('zkShuffle State Less Unit Test', function () {
+    let sm_owner : SignerWithAddress
+    let game_owner : SignerWithAddress
+    let signers : SignerWithAddress[]
+	before(async () => {
+        signers = await ethers.getSigners()
+        sm_owner = signers[10];
+        game_owner = signers[11];
+	});
+
+    it('Player Register StateLess', async () => {
+        const SM = await deploy_shuffle_manager(sm_owner)
+        const gameId = 1
+        const numCards = 5
```

Comment:
should add tests for 52 cards

Reply:
Yeah, better to add more test coverage in new PR for move fast forward here.